.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,18 @@
+name: CI_pull_request
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Ballerina Build
+        uses: ballerina-platform/ballerina-action/@nightly
+        with:
+          args:
+            build -c --skip-tests
```

Comment:
No need to skip tests

Reply:
Ballerina platform secrets cannot be accessed by pull request job. So we can skip the tests here.